Lesson14/Akan/script.js

```diff
@@ -0,0 +1,192 @@
+  console.log("Çarpım sonucu= " + carpim);
+  console.log("Bölüm sonucu= " + bolum);
+
+  // 4. Use template literals to log: "My name is (userName). I like JS."
+  // CODE HERE
+  console.log(`My name is ${userName}. I like JS.`);
+
+  // 5. Declare a string password = "securePass". Log the length of password.
+  // CODE HERE
+  const password = "securePass";
+  console.log(password.length);
+
+  // 6. Convert the string "hello world" to uppercase and log it.
+  // CODE HERE
+  const word = "hello world";
+  console.log(word.toUpperCase());
+  console.log("hello world".toUpperCase());
+
+  // 7. Concatenate "Hello" and "World" with a space in between and log the result.
+  // CODE HERE
+
+  // 8. Check the type of a variable, e.g., let x = 42. Log the type using typeof.
+  // CODE HERE
+  const name = "sevcan";
+  const city = "mersin";
+
+  console.log(typeof name, typeof city, typeof userAge);
+
+  // 9. Convert the number 100 to a string and log the result.
+  // CODE HERE
+  const number = 100;
+  const numberString = number.toString();
+  console.log(number, numberString);
+  // This will create a string from a number and a string!!! console.log(number + numberString);
+
+  // 10. Convert the string "50" to a number and log its type to confirm the conversion.
+  // CODE HERE
+  const string = "50";
+
+  console.log(parseInt(string));
+
+  // 11. Generate a random integer between 0 and 10 and log it.
+  // CODE HERE
+  const randomNumber = Math.random() * 11; // 0 -> 10.99999
+  console.log('Random number ex. 11', Math.floor(randomNumber));
+
+  // 12. Round the number 3.7 down using Math.floor and 3.2 up using Math.ceil, log both.
+  // CODE HERE
+  const number1Ex12 = Math.floor(3.7);
+  console.log(number1Ex12);
+  const number2Ex12 = Math.ceil(3.2);
+  console.log(number2Ex12);
+
+
+  // Prompt can only be used in browser environment
+  const userResponse = prompt("What is your age?");
+  const userResposeNumber = parseInt(userResponse);
+
+  if(isNaN(userResposeNumber)) {
+    throw new Error("Please enter a valid number");
+} else {
+  console.log('userResponse', Math.floor(userResposeNumber));
+}
+
+// 13. Declare a boolean variable isStudent = true. Log it.
+// CODE HERE
+const isStudent = true;
+console.log(isStudent);
+
+
+// 14. Initialize counter = 0, then increment it by 1 using counter++ and log it.
+// CODE HERE
+let counter = 0;
+counter++;
+console.log(counter);
+
+// 15. Initialize points = 10, add 5 to it using points += 5, then log points.
+// CODE HERE
+let points = 10;
+points += 5;
+console.log(points);
+
+// 16. Declare name="Alice", age=30, city="Paris". Log "Alice (30) lives in Paris" using template literals.
+// CODE HERE
+const name = "Alice";
+const age = 30;
+const city = "Paris";
+console.log('${name}', '(${age}', 'lives in ${city}';
+
+
+// 17. Declare variables x=5, y=10, z=15. Log their total sum.
+// CODE HERE
+const x = 5;
+const y = 10;
+const z = 15;
+console.log(x + y + Z);
```

Comment:
JS is a case-sensitive language. The variable Z should be lowercase z.
```suggestion
console.log(x + y + z);
```